code/datums/wounds/slash.dm

```diff
@@ -30,6 +37,20 @@
 	/// A bad system I'm using to track the worst scar we earned (since we can demote, we want the biggest our wound has been, not what it was when it was cured (probably moderate))
 	var/datum/scar/highest_scar
 
+//ok so this is ultra stupid, but we need to update examine and shit in case the owner is not a bleeder
+/datum/wound/slash/apply_wound(obj/item/bodypart/L, silent, datum/wound/old_wound, smited, attack_direction)
```

Comment:
Can you just make a seperate proc for this and put it on the wounds parent? Seeing this copypasted everywhere is not great

Reply:
I can generalize var/no_bleeding for the wound parent, but it still won't solve most of the issues besides for this proc